docs/degree_programs_search_shortcode.md

```diff
@@ -28,6 +29,13 @@ See below for a list of supported attributes.
 | `subject-group`         | Subject group filter                                                    | Comma-separated term names                        |
 | `teaching-language`     | Teaching language filter                                                | Comma-separated term names                        |
 
+## List of elements that can be excluded from display
```

Comment:
I 'd suggest avoiding the use of the term "exclude" since this is an attrubute of another shortcode (single degree program) and may cause confusion.

```suggestion
## List of elements that can be hidden
```